src/PortToTripleSlash/src/libraries/RoslynTripleSlash/TripleSlashSyntaxRewriter.cs

```diff
@@ -501,38 +507,65 @@ private static SyntaxTriviaList GetLeadingTrivia(SyntaxNode node)
             return new();
         }
 
-        private static SyntaxTriviaList GetSummary(DocsAPI api, SyntaxTriviaList leadingWhitespace)
+        // Collects all tags with of the same name from a SyntaxTriviaList.
+        private static SyntaxTriviaList FindTag(string tag, SyntaxTriviaList leadingWhitespace, SyntaxTriviaList from)
+        {
+            List<XmlNodeSyntax> l = new();
+            foreach(var trivia in from)
+            {
+                SyntaxNode? structure = trivia.GetStructure();
+                if (structure is DocumentationCommentTriviaSyntax st) {
```

Comment:
Since you're not using `structure` anywhere elser, but you _do_ use `st`, I think it's better to merge them, and use `structure` instead of `st`:
```suggestion
                if (trivia.GetStructure() is DocumentationCommentTriviaSyntax structure) {
```